omnigibson/object_states/particle_modifier.py

```diff
@@ -240,13 +240,6 @@ def is_compatible(cls, obj, **kwargs):
 
         return True, None
 
-    def initialize_link_mixin(self):
-        # Run super first
-        super().initialize_link_mixin()
-
-        # Make sure there's at most only a single metalink
-        assert len(self.links) <= 1, f"Can only have at most a single metalink for {self.__class__.__name__}!"
-
```

Comment:
why remove?

Reply:
there's objects that have multiple sources (e.g.: sink with two faucets), and I couldn't remember why I implemented this constraint originally.

Of course, the current code only functionally supports a single link anyways, but that's the same for all of our object states -- we need to handle multi-links properly anyways, it's a future todo

Reply:
Got it!